README.md

```diff
@@ -170,10 +170,10 @@ val topLevelSubscriberActor: ActorRef = kafka.producerActor(producerProperties)
 ```
 
 #### Handling errors
-When a consumer or a producer fail to read/write from Kafka, the error is unrecoverable and it requires that
-the connection get terminated. This will be performed automatically and the KafkaActorSubscriber / KafkaActorPublisher
-which failed will be stopped. You can use DeathWatch to detect such failures in order to restart your stream.
-Additionally, when a producer fails, it will signal `onError()` to its inputs in order to stop the rest of stream 
+When a consumer or a producer fails to read/write from Kafka, the error is unrecoverable and requires that
+the connection be terminated. This will be performed automatically and the `KafkaActorSubscriber` / `KafkaActorPublisher`
+which failed will be stopped. You can use `DeathWatch` to detect such failures in order to restart your stream.
+Additionally, when a producer fails, it will signal `onError()` to its inputs <!-- "inputs"? You mean it signals its failure upstream, is this true? Maybe other wording for people like me to get it? :) AKa060216 --> to stop the rest of stream 
```

Comment:
I don't get the "it will signal onError() to its inputs in order to stop the rest of the stream". If you were so kind and explain to me this a bit more in detail, I could see if the text can be modified so I would have understood it right up.